test/test_account.ts

```diff
@@ -20,5 +21,6 @@ export const test_special_account: OSMLAccount = {
   region: "us-isob-east-1",
   isAdc: true,
   deployModelRunner: true,
-  deployTileServer: true
+  deployTileServer: true,
+  deployDataIntake: true
```

Comment:
Could you update the README.md in `lib/accounts` in the guidance repo to reflect on this?

Reply:
Will do in the Guidance PR